ci/pipeline.yml

```diff
@@ -132,26 +95,18 @@ jobs:
 
 resources:
 - name: common-staging
-  type: cg-common
+  type: s3-iam
   source:
-    bucket_name: {{admin-ui-private-bucket-staging}}
-    secrets_files:
-    - admin-ui.main.yml
-    - admin-ui.external.yml
-    secrets_passphrase: {{admin-ui-staging-private-passphrase}}
-    bosh_cert: bosh.pem
-    region: {{aws-region}}
+    bucket: {{admin-ui-private-bucket-staging}}
+    versioned_file: staging-admin-ui-decrypted.yml
```

Comment:
There were two files used before, is this accounted for with this one file?

Reply:
Yes.  I couldn't see a reason why we had two, so I mushed them into one to make it more standard.  I can't see any reason why it would break stuff, but if you do, let me know!